Get orders

olorunwalawrence opened the pull request:
#### What does this PR do?
create get order functionality
#### Description of the task to be completed
- get all order functionality
- get all order routes
- integrate mentor feed back
#### How to manually test this PR
- clone the [repo ](https://github.com/olorunwalawrence/book-a-meal)
- on your terminal, run `git checkout  get-orders-#163848738`
- on your terminal, run npm install to install dependencies
- on your terminal, run npm run start:dev to start the application.



#### What is the story identity?

 #163848738

olorunwalawrence committed:
ft(menu):setup meu
 - create menu controller function
- menu route path
- validate menu field

[Delivers #163848726]

olorunwalawrence committed:
ft(menu):get all menu
- create menu functionality
- create menu route
- create menu dummy database
[Delivers #163848732]

olorunwalawrence committed:
ft(order): mkeOrder
- create order database object
- create order functionality
- create order route

[Delivers #163848734]

olorunwalawrence committed:
ft(order): update order
- create order functionality
- create order route
- check menu availability
[Delivers #163848744]

olorunwalawrence committed:
ft(getall):getllorders
- get all orders
- create get all functionality
- get all routes
[Delivers #163848738]

olorunwalawrence committed:
ft(feedback):
- integrate mentor feed back

Billmike commented on dummy/controllers/orderController/orderController.js:
```diff
@@ -0,0 +1,79 @@
+
+
+    const data = {
+      id: orderDb.length + 1,
```
You can use shortId here as well

Billmike commented on app.js:
```diff
@@ -23,3 +23,5 @@ app.route('/*').all((req, res) => res.status(404).json({
   console.log('server is up and running');
 });
+
+export default app;
```
Add a new line here

Billmike commented on dummy/controllers/mealController/mealController.js:
```diff
@@ -22,10 +24,14 @@ export default class Meal {
     );
     if (!result.length < 1) {
-      return res.status(400).json({ message: 'Meal title already exist' });
+      return res.status(400).json({
```
Response status should be 409

Billmike commented on dummy/controllers/menuController/menuController.js:
```diff
@@ -0,0 +1,58 @@
+    // PUSH DATA INTO DUMMY DATABASE
+
+    menuDb.push(data);
+    return res.status(200).json({
```
Response status should be 201. [See this link](https://httpstatuses.com/) for http statuses and their meanings

olorunwalawrence committed:
ft(update): implement mentor feed back
- use shortid to generate unique id,
- update status code
- eslint new line
[Delivers #163848738 ]

Billmike reviewed (approved):
Looks good
